package.json

```diff
@@ -66,7 +66,7 @@
     "mongoose": "^4.2.7",
     "nconf": "^0.7.1",
     "nexus-node": "^0.4.21",
-    "nexus-warp": "0.0.6",
+    "nexus-warp": "git+https://github.com/DaQuirm/nexus-warp.git",
```

Comment:
Yes, releases became too frequent :smile_cat: 
I think it's more practical to keep it like that for a while